### PR TITLE
Restrict remove permissions from group

### PR DESCRIPTION
**Pull request by IKarbowiak**
User shouldn't be allowed to remove any permission from the group.
Prevent removing permissions from the group which left not manageable permissions.

# Impact

* [ ] New migrations
* [x] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [x] The changes are tested
* [x] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

File: saleor/graphql/account/mutations/permission_group.py [hunk 1]
```diff
@@ -88,13 +89,16 @@ def clean_input(
...
         requestor: "User",
+        group: auth_models.Group,
         errors: Dict[Optional[str], List[ValidationError]],
         cleaned_input: dict,
     ):
         field = "add_permissions"
         permission_items = cleaned_input.get(field)
         if permission_items:
             cleaned_input[field] = get_permissions(permission_items)
+            if requestor.is_superuser:
```
Comment (fowczarek) [inline saleor/graphql/account/mutations/permission_group.py:100]: Maby `if not requestor.is_superuser" ?

File: saleor/graphql/account/mutations/permission_group.py [hunk 2]
```diff
@@ -244,17 +246,39 @@ def ensure_requestor_can_manage_group(
...
         errors: Dict[Optional[str], List[ValidationError]],
         cleaned_input: dict,
     ):
-        super().clean_permissions(requestor, errors, cleaned_input)
+        super().clean_permissions(requestor, group, errors, cleaned_input)
         field = "remove_permissions"
         permission_items = cleaned_input.get(field)
         if permission_items:
             cleaned_input[field] = get_permissions(permission_items)
+            if requestor.is_superuser:
```
Comment (fowczarek) [inline saleor/graphql/account/mutations/permission_group.py:258]: Maby `if not requestor.is_superuser" ?

File: saleor/graphql/account/mutations/permission_group.py (hunk 1 above)
Comment (IKarbowiak) [reply saleor/graphql/account/mutations/permission_group.py:100]: Why? With opposite condition, it'll be more nesting.

Comment (fowczarek) [reply saleor/graphql/account/mutations/permission_group.py:100]: To not use empty 'return' :D

Comment (maarcingebala) [reply saleor/graphql/account/mutations/permission_group.py:100]: Agree, this would look a bit better.

**Pull request merged**